src/stp/new_plays/AggressiveFormation.cpp

```diff
@@ -25,7 +25,7 @@ AggressiveFormation::AggressiveFormation() : Play() {
         std::make_unique<role::BallAvoider>(role::BallAvoider("offender_3"))};
 }
 
-uint8_t AggressiveFormation::score(world_new::World* world) noexcept { return 2; }
+uint8_t AggressiveFormation::score(world_new::World* world) noexcept { return 100; }
```

Comment:
Make a constant class variable for 100 and explain the meaning of it.

Reply:
No because the scoring will be fixed in a future PR